src/api.js

```diff
@@ -54,9 +58,18 @@ window.api = {
     window.application.$store.dispatch('project/setSpacing', { spacing: window.api.config.initialGridSize });
     window.application.$store.dispatch('project/setNorthAxis', { north_axis: window.api.config.initialNorthAxis });
 
-    // if the map modal has been disabled, mark the map as initialized so that time travel can be initialized
-    // TODO: we may want to intitialize timetravel in the importFloorplan action instead
-    window.application.$store.dispatch('project/setMapInitialized', { initialized: true });
+    if (window.api.config.showMapDialogOnStart) {
+      window.application.$store.dispatch('project/setMapEnabled', { enabled: true });
+      window.application.$store.dispatch('project/setMapVisible', { visible: true });
+      window.application.$store.dispatch('project/setMapInitialized', { initialized: false });
+    } else {
+      window.application.$store.dispatch('project/setMapEnabled', { enabled: false });
+      window.application.$store.dispatch('project/setMapVisible', { visible: false });
+      window.application.$store.dispatch('project/setMapInitialized', { initialized: false });
+    }
```

Comment:
potential refactor:
```
window.application.$store.dispatch('project/setMapEnabled', { enabled: window.api.config.showMapDialogOnStart });
window.application.$store.dispatch('project/setMapVisible', { visible: window.api.config.showMapDialogOnStart });
window.application.$store.dispatch('project/setMapInitialized', { initialized: false });
```

(instead of the if stmt)